tests/PHPStan/Analyser/data/bug-10002.php

```diff
@@ -0,0 +1,26 @@
+<?php declare(strict_types=1);
+
+namespace Bug100002;
+
+use function PHPStan\Testing\assertType;
+
+class Foo
```

Comment:
does this test cover bother changed execution paths?

Reply:
Good point - it doesn't. I'll add a test for the other direction as well.